staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -90,25 +83,6 @@ var (
 		},
 		[]string{"verb", "dry_run", "group", "version", "resource", "subresource", "scope", "component"},
 	)
-	deprecatedRequestLatencies = prometheus.NewHistogramVec(
-		prometheus.HistogramOpts{
-			Name: "apiserver_request_latencies",
-			Help: "(Deprecated) Response latency distribution in microseconds for each verb, group, version, resource, subresource, scope and component.",
-			// Use buckets ranging from 125 ms to 8 seconds.
-			Buckets: prometheus.ExponentialBuckets(125000, 2.0, 7),
-		},
-		[]string{"verb", "group", "version", "resource", "subresource", "scope", "component"},
-	)
-	deprecatedRequestLatenciesSummary = prometheus.NewSummaryVec(
```

Comment:
Our scalability tests are still relying on that metric.
Can we please leave that for a bit more time until we migrate tests to it?

[It's a bunch of non-trivial work, that has been started at the beginning of the release, but still is not finished.]

@mm4tt @krzysied @mborsz

Reply:
after offline discuss with @brancz .
If there is an explicit reason and a timeline to fix the dependencies and eventually remove them, we agree to keep the old ones.

Reply:
@wojtek-t which metrics are scalability tests relying on ?
Just this one `apiserver_request_latencies_summary` ?

Reply:
Yes - we just need `apiserver_request_latencies_summary`.
We're already working on no-longer-relying on those and we have it working. It's mostly ensuring that it is stable, works-as-expected and potentially some bug fixing.
So I really expect we will be able to remove that in 1.16 (hopefully in the relatively early phase of that).

Reply:
thanks.
Bring it back in https://github.com/kubernetes/kubernetes/pull/76496/commits/2b7353f48008567ce10ea671bef5114ca71de1aa. PTAL

Reply:
Thanks a lot - this looks good to me now.